learn/multi_search/implement_sharding.mdx

```diff
@@ -11,6 +11,10 @@ Sharding is the process of splitting an index containing many documents into mul
 
 This guide walks you through activating the `/network` route, configuring the network object, and performing remote federated searches.
 
+<NoticeTag label="Enterprise Edition">
+Sharding is an Enterprise Edition feature. You are free to use for evaluation purposes. Please [reach out to us](mailto:sales@meilisearch.com) before using it in production.
```

Comment:
@qdequele do we want cloud users to reach sales too? or is it "free" for Cloud users. I do not expect a lot to try it right now, but can happen in a near future. They will pay "automatically" more by having more machines for the moment

Reply:
Check with @qdequele: @dureuill it's ok as it is 👍